tests/tickets/gh1160/1.html

```diff
@@ -0,0 +1,80 @@
+<head>
```

Comment:
Can't these tests be moved to `plugins/widget`? They're widget specific, so that place seems more obvious to me. We got even `plugins/widget/integration` directory.

Reply:
👍  Moved to `widget` directory.